src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1113,8 +1113,19 @@ public void updateInstallation(int id, String json) {
                 logger.debug("Request {} {}", request.getMethod(), request.getUrl());
             }
             return client.executeRequest(request, handler);
```

Comment:
catching throwable here as 1.9.3 no longer throws IOException so we can support 1.7.12 (which anyway caught everything and wrapped it in IOException).